README.md

```diff
@@ -19,11 +34,148 @@ helm install stunner-gateway-operator stunner/stunner-gateway-operator --create-
+Caveats: 
+- currently the max lifetime for `stunner` to remain alive after deleted in 1 hour: this means that `stunner` will remain active only for 1 hour after it has been deleted/scaled-down. You can always set this in by adjusting the `terminationGracePeriod` on your `stunnerd` pods.
+- if there are active (or very recent) TURN allocations then the `stunner` pod may refuse to be removed after a kubectl delete. Use `kubectl delete pod --grace-period=0 --force -n stunner stunner-XXX` to force removal.
+
+The default termination period is set to 3600 seconds (1 hour). To modify it use the `--set stunner.deployment.container.terminationGracePeriodSeconds=<NEW_PERIOD_IN_SECONDS>` flag.
+
+### Monitoring
+
+The monitoring guide assumes an up-and-running `Stunner-Gateway-Operator` in the user's Kubernetes cluster.
+
+#### Configuration
+
+Metrics collection is *not* enabled in the default installation. In order to open the
+metrics-collection endpoint for a [gateway hierarchy](https://github.com/l7mp/stunner/blob/main/docs/GATEWAY.md), configure an
+appropriate HTTP URL in the `metricsEndpoint` field of corresponding the
+[GatewayConfig](https://github.com/l7mp/stunner/blob/main/docs/GATEWAY.md#gatewayconfig) resource.
+
+For instance, the below GatewayConfig will expose the metrics-collection server on the URL
+`http://:8080/metrics` in all the STUNner media gateway instances of the current gateway hierarchy.
+
+```yaml
+apiVersion: stunner.l7mp.io/v1alpha1
+kind: GatewayConfig
+metadata:
+  name: stunner-gatewayconfig
+  namespace: stunner
+spec:
+  userName: "my-user"
+  password: "my-password"
+  metricsEndpoint: "http://:8080/metrics"
+```
+
+#### Installation
+
+A full-fledged Prometheus+Grafana helm chart is available in the repo [here](/helm/stunner-prometheus/). To use this chart, install the Prometheus+Grafana stack with helm.
+
+1. **[Expose the STUNner metrics-collection server in the GatewayConfig](#configuration)**
+
+
+2. **Install the Prometheus+Grafana stack with a helm chart**
+
+This helm chart creates a ready-to-use Prometheus+Grafana stack in the `monitoring` namespace: installs Prometheus along with the prometheus-operator, and Grafana; configures PodMonitor for monitoring STUNner pods, and sets up Prometheus as a datasource for Grafana.
+
+```console
+helm repo add stunner https://l7mp.io/stunner
+helm repo update
+
+helm install prometheus stunner/stunner-prometheus
+```
+
+Learn more about how to enable, configure and use monitoring [here](https://github.com/l7mp/stunner/blob/main/doc/MONITORING.md#installation).
+
+#### Disabling monitoring
+
+The default is to deploy STUNner with the monitoring port exposed in the STUNner pods to enable metric collection on K8s networking level.
+
+In order to disable monitoring install the STUNner chart with the following command.
```

Comment:
If you wish to avoid the port exposition, you can install STUNner with monitoring disabled: